### ISSUE TITLE
Document whether ER20 tokens are supported

### ISSUE THREAD

**Issue by ripper234** [christianlundkvist/simple-multisig#29]
Hi,

We tried to contact you but couldn't find your email, and you didn't respond to a tweet we sent you a week ago.

We are looking into this contract for production purposes, would like to understand a few things. Specifically, my question right now is: does the contract support ER20 tokens? If yes or no, perhaps you can add a comment about it in the readme, or if it doesn't support, perhaps leave this issue open as a TODO?

In any case we would be interested in discussing further, would appreciate it if you dropped me a line at ron.gross@gmail.com

**Comment by christianlundkvist** [comment]
Hi @ripper234! Since the contract supports any function call with arbitrary data it will also support handling ERC20 tokens. The linked Medium article states that ERC20 tokens are supported but I can also add something in the README.

**Comment by ripper234** [comment]
Sounds good. I guess I need to dig a bit in order to understand how to actually run it with ERC20 tokens, I've been using [the UI](https://www.reddit.com/r/ethereum/comments/82pyi9/i_made_a_front_end_for_christian_lundkvists/) for the time being which for now doesn't support ERC20.

**Comment by christianlundkvist** [comment]
@ripper234 Yeah you'd need to have the destination address be the token contract address, the value sent be 0, and the data field would encode that you are sending X tokens from A to B.
